README.md

```diff
@@ -430,8 +432,6 @@ For information about the governance of the Node.js project, see
 **Thorsten Lorenz** &lt;thlorenz@gmx.de&gt;
 * [TimothyGu](https://github.com/TimothyGu) -
 **Tiancheng "Timothy" Gu** &lt;timothygu99@gmail.com&gt; (he/him)
-* [tniessen](https://github.com/tniessen) -
-**Tobias Nießen** &lt;tniessen@tnie.de&gt;
```

Comment:
@mhdawson - I guess this (collaborator listing) needs to be retained?

Reply:
I'll double check but I believe we only list in either the TSC list or collaborators but not both

Reply:
You are correct, I thought I'd checked that we just moved, but I see I got it wrong.